kernelci/build.py

```diff
@@ -1115,6 +1129,10 @@ def run(self, jopt=None, verbose=False, opts=None):
             'publish_path': publish_path,
         }
 
+        if target.startswith("cip://"):
+            self._create_cip_config(target)
+            target = "olddefconfig"
+
         res = self._make(target, jopt, verbose, opts)
```

Comment:
I think it's better to not change the value of `target` as it's coming from the input options.  Instead, we could do this:
```suggestion
        if target.startswith("cip://"):
            self._create_cip_config(target)
            res = self._make('olddefconfig', jopt, verbose, opts)
        else:
            res = self._make(target, jopt, verbose, opts)
```